<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="inicio_adm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>INICIO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cpf_adm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conta_nova">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CONTA NOVA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="avisos_adm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AVISOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="criacao_adm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CRIAÃ‡AO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carros_adm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CARROS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
